common/src/scheduler/task.rs

```diff
@@ -109,7 +111,7 @@ pub trait TaskHandler<AccountId> {
     fn send_dispatch(&mut self, stashed_message_id: MessageId);
 
     // Send delayed message to user action.
-    fn send_user_message(&mut self, stashed_message_id: MessageId);
+    fn send_user_message(&mut self, stashed_message_id: MessageId, is_mailboxed: bool);
```

Comment:
it's not already mailboxed, this argument is about should the message be mailboxed once sent after delay

Reply:
done